docs/iris/example_code/Meteorology/wind_speed.py

```diff
@@ -0,0 +1,77 @@
+"""
+Plotting wind direction using quiver
+===========================================================
+
+This example demonstrates using quiver to plot wind speed
```

Comment:
contours and wind direction arrows from wind vector component input data.  The vector components are co-located in space in this case.